client/state/happychat/selectors/can-user-send-messages.js

```diff
@@ -9,6 +8,14 @@ import {
 import getHappychatChatStatus from 'calypso/state/happychat/selectors/get-happychat-chat-status';
 import isHappychatClientConnected from 'calypso/state/happychat/selectors/is-happychat-client-connected';
```

Comment:
you might want to relativize these imports to `./get-happychat-chat-status` etc.

Reply:
Sure, done in https://github.com/Automattic/wp-calypso/pull/60043/commits/2f8bd314086cc200a4893035505a6b2189f0ccbb